content/applicants/student-life.md

```diff
@@ -0,0 +1,20 @@
+title: О студенческой жизни
+slug: student-life
+category: applicants
+tags: [_nav,_ignore_nav]
```

Comment:
Add `_category_header` to the `tags` to display category header image. Remember to delete the image.